fuel-asm/src/op.rs

```diff
@@ -312,7 +342,12 @@ const _: () = {
 
     #[wasm_bindgen::prelude::wasm_bindgen]
     /// Construct a `WQDV` instruction from its arguments.
-    pub fn wqdv_args(ra: u8, rb: u8, rc: u8, args: DivArgs) -> typescript::Instruction {
+    pub fn wqdv_args(
+        ra: u8,
+        rb: u8,
+        rc: u8,
+        args: wideint::DivArgs,
+    ) -> typescript::Instruction {
         crate::op::wqdv_args(ra, rb, rc, args).into()
     }
```

Comment:
We should also add exports for the new instructions here to support them in the TS-SDK.

Reply:
I don't think the exports are missing? They're automatically exported by 

```
pub use super::_op::*;
```

(and I see them in rustdoc generated by `cargo doc -p fuel-asm --features typescript`)

Reply:
Ah I see it now, looks like these are just special helpers for a select few opcodes but not required for all of them 👍🏻